linux/switchtec.h

```diff
@@ -138,17 +138,21 @@ enum {
 };
 
 enum {
-	SWITCHTEC_GEN4_BL2_0_RUNNING = 0x01,
-	SWITCHTEC_GEN4_BL2_1_RUNNING = 0x02,
-	SWITCHTEC_GEN4_MAP0_RUNNING = 0x03,
-	SWITCHTEC_GEN4_MAP1_RUNNING = 0x04,
-	SWITCHTEC_GEN4_CFG0_RUNNING = 0x05,
-	SWITCHTEC_GEN4_CFG1_RUNNING = 0x06,
-	SWITCHTEC_GEN4_IMG0_RUNNING = 0x07,
-	SWITCHTEC_GEN4_IMG1_RUNNING = 0x08,
+	SWITCHTEC_GEN4_MAP0_RUNNING = 0x00,
+	SWITCHTEC_GEN4_MAP1_RUNNING = 0x01,
+	SWITCHTEC_GEN4_KEY0_RUNNING = 0x02,
+	SWITCHTEC_GEN4_KEY1_RUNNING = 0x03,
+	SWITCHTEC_GEN4_BL2_0_RUNNING = 0x04,
+	SWITCHTEC_GEN4_BL2_1_RUNNING = 0x05,
+	SWITCHTEC_GEN4_CFG0_RUNNING = 0x06,
+	SWITCHTEC_GEN4_CFG1_RUNNING = 0x07,
+	SWITCHTEC_GEN4_IMG0_RUNNING = 0x08,
+	SWITCHTEC_GEN4_IMG1_RUNNING = 0x09,
```

Comment:
Ugh, who lets the firmware people make unnecessary incompatible changes? That's a terrible practice. If they had simply added the key as 0x9 and 0xA this wouldn't be necessary and there wouldn't exist firmware and software that are now incompatible. Absolutely gross.

Reply:
It happens before RTP, though we can definitly avoid it. I'd expect they will keep it compatible afterward.